service_thing_aggregate/src/main/resources/evolutions/default/1.sql

```diff
@@ -23,3 +23,13 @@ CREATE TABLE MUSARK_STORAGE.STORAGE_NODE (
  museum_id INTEGER NOT NULL,
  PRIMARY KEY (storage_node_id)
 );
+
+CREATE TABLE MUSIT_MAPPING.MUSITTHING (
```

Comment:
I've added the table to postgres. (Hopefully I'll remember to ask Svein or Rituvesh to help me start the stuff, to check that I didn't mess up the init.sql file!)
Natural history people use the term 'Object'. The Cultural side don't use that term and prefer 'Gjenstand'. 'Thing' is sort of a compromise. It is also what CIDOC CRM has ended up using.